Sources/Model/TransferMethod/HyperwalletBankCard.swift

```diff
@@ -24,7 +24,7 @@ public class HyperwalletBankCard: HyperwalletTransferMethod {
     override private init(data: [String: AnyCodable]) {
         super.init(data: data)
     }
-
+    /// Initialization of HyperwalletBankCard
```

Comment:
update to - /// The required initializer

Reply:
Updated